AU2/plugins/util/game.py

```diff
@@ -35,4 +35,14 @@ def soft_escape(string: str) -> str:
     # and remember that code injection without explicit consent is illegal (CMA sxn 2/3)
     if not string.startswith("<!--HTML-->"):
         return escape(string)
-    return string
\ No newline at end of file
+    return string
+
+
+def escape_format_braces(string: str) -> str:
+    """
+    Escapes { and } in a string so that they will be processed correctly by .format
+    This needs to be called on user-input strings passed as part of the message of any inquirer prompt,
+    or as part of the default of inquirer Text prompts (but not the choices or defaults of List of Checkbox prompts)
+    """

```

Comment:
Can you finish the doc-string with params & return? I need to clean up the rest of this file - most of it dates back to the days before MW24 where speed was needed over style.

My preferred style is [Google's doc string style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html), e.g.:

```
def function_with_types_in_docstring(param1, param2):
    """Example function with types documented in the docstring.

    `PEP 484`_ type annotations are supported. If attribute, parameter, and
    return types are annotated according to `PEP 484`_, they do not need to be
    included in the docstring:

    Args:
        param1 (int): The first parameter.
        param2 (str): The second parameter.

    Returns:
        bool: The return value. True for success, False otherwise.

    .. _PEP 484:
        https://www.python.org/dev/peps/pep-0484/

    """


def function_with_pep484_type_annotations(param1: int, param2: str) -> bool:
    """Example function with PEP 484 type annotations.

    Args:
        param1: The first parameter.
        param2: The second parameter.

    Returns:
        The return value. True for success, False otherwise.

    """
```